zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -142,6 +147,22 @@ function PageWrapper(props) {
     [],
   );
 
+  //Logic behind showing and hiding scroll to top button
+  const handleScroll = ()=>{
+    if(window.pageYOffset > 250){
```

Comment:
How did we decide on the 250 value here?

Reply:
I thought about it and tested it once again and i found it is a very large gap and button gets displayed very late. So, I think we should keep it 70, at 70 scroll back to top button will be displayed when scrollbar crosses more than half of the navbar. I am open to suggestions.